DoLah/Commands/Command.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include "..\Models\Task.h"
+
+namespace DoLah {
+
+	class Calendar;
+
+	class AbstractCommand {
```

Comment:
We need this to be an `ITaskCommand` as an interface, not an abstract class of `AbstractCommand`.